.github/workflows/create-draft-release-reusable.yml

```diff
@@ -0,0 +1,43 @@
+name: Create draft release (reusable)
+
+on:
+  workflow_call:
+    inputs:
+      VERSION:
+        required: true
+        type: string
+        description: The semantic version number.
+    secrets:
+      BOT_PAT:
```

Comment:
(nit) for readability, does it make sense to rename `BOT_PAT` to `BOT_TOKEN`?

Reply:
Mhh, I am indifferent to the wording. My argument for PAT is that the personal access token is special kind of token. But `BOT_TOKEN` would be fine for me too.

Reply:
What do you say?

Reply:
It is a nitpick, let's keep the pat!